lib/syntax_tree/node.rb

```diff
@@ -2194,23 +2211,23 @@ def call(q)
     SEPARATOR = Separator.new.freeze
 
     def format(q)
-      q.text("|")
-      q.group do
-        q.remove_breaks(q.format(params))
+      pipe? ? q.remove_breaks(q.format(params)) : q.format(params)
```

Comment:
I managed to move the first two lines of this method into `BlockNode`, but I"m having trouble with `q.remove_breaks` because the tests seem to expect that all params within pipes are on the same line. Any suggestions?